packages/blocks/src/_legacy/content-parser/index.ts

```diff
@@ -996,6 +996,15 @@ export class ContentParser {
         options: tags,
       },
     });
+
+    // Make sure the title is synced with the model
+    const pageMetaTitle = this._page.meta.title;
+    const pageModelTitle = (this._page.root as PageBlockModel).title.toString();
```

Comment:
Could you please try something like `if (this._page.root instanceof PageBlockModel)` to guard this?